{{cookiecutter.project_slug}}/frontend/app/components/Footer.tsx

```diff
@@ -0,0 +1,108 @@
+"use client"
+
+import Link from "next/link"
+
+const siteName: string = "CompanyName"
```

Comment:
Should this be something replaced by the cookiecutter?

```suggestion
const siteName: string = "{{ cookiecutter.project_name }}"
```